lib/symbol/read.c

```diff
@@ -441,8 +441,8 @@ SYMBOL *S_read(const char *sname)
 	    }
 	}
 	else {
-	    sprintf(buf, "Unknown keyword in symbol: '%s'", buf);
-	    return (err(fp, symb, buf));
+	    sprintf(buf2, "Unknown keyword in symbol: '%s'", buf); //
```

Comment:
```suggestion
	    sprintf(buf2, "Unknown keyword in symbol: '%s'", buf);
```
Please no `//` comments in C code.